### PR TITLE
Labels react

### PR DESCRIPTION
**Pull request by anna-wro**
screen shot 2017-07-12 at 12 53 57

### CODE REVIEW

File: src/components/labels/Label.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      <div className="sg-label__number">{number}</div>
+    </div>;
+  }
+
+  return <div className={labelClass}>
+    <div className="sg-label__icon">
+      <Icon type={iconType} color={iconColor} size={size.ICON_SIZE}/>
+    </div>
+    <div className="sg-label__text">{text}</div>
+  </div>;
```
Comment (kdzwinel) [inline src/components/labels/Label.jsx]: this code is almost the same as above - maybe we can do sth like {name ? <div>bla</div> : null}. @matikrk @aju WDYT?

File: src/components/labels/Label.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+  const labelClass = classNames('sg-label', {
+    [`sg-label--${size.CLASS_NAME}`]: size !== SIZE.NORMAL,
+    'sg-label--secondary': secondary,
+    'sg-label--unstyled': unstyled,
+    'sg-label--emphasised': emphasised,
+    'sg-label--elements-to-the-top': elementsToTop
+  });
+
+  if (number) {
```
Comment (matikrk) [inline src/components/labels/Label.jsx]: better use variable 
```
let numberElement;
if(number){
numberElement=     <div className="sg-label__number">{number}</div>;
}
```

**Comment by anna-wro** [comment]
@kdzwinel, @matikrk - I changed it

File: src/components/labels/Label.jsx (hunk 2 above)
Comment (anna-wro) [inline src/components/labels/Label.jsx]: Thanks!

File: src/components/labels/Label.jsx (hunk 1 above)
Comment (anna-wro) [inline src/components/labels/Label.jsx]: I did what @matikrk suggested

**Pull request merged**